diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5ad2cc10..8763631e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1613,7 +1613,7 @@ if(BUILD_SHARED)
       proxy_thr)
 
   if(NOT CMAKE_BUILD_TYPE STREQUAL "Debug") # Why?
-    option(WITH_PERF_TOOL "Build with perf-tools" ON)
+    option(WITH_PERF_TOOL "Build with perf-tools" OFF)
   else()
     option(WITH_PERF_TOOL "Build with perf-tools" OFF)
   endif()
